KB migration: Change New Relic account owner (#5072)
magento/devdocs: guides/v2.2/cloud/project/new-relic.md

-  The New Relic worst-case requests are not averages, but edge cases; whereas, Blackfire provides more of the average worst-case requests.
-  Blackfire is not detailed in determining and displaying queries giving you trouble, but you can use New Relic to find those.
-  Background processes, like Cron job, can cause issues. These background processes are not monitored by New Relic, but you can watch them using Blackfire.

## Change account owner

> See [New Relic's documentation][1].

Reviewer: Why? Seems early for this.

Author: The idea is to immediately direct readers New Relic's docs to learn how to do this. I didn't feel like a long explanation was necessary.

> Make sure the user that you want to assign the Owner role to is listed on your {{site.data.var.ece}} account. See [Add a new user][2]. If a user needs to be added, but no existing Owner or Admin on the account can help, any Magento user who has access to New Relic's [Magento Partnership Owner Account][3] can add users on the customer's behalf.

Reviewer: ```suggestion
Before assigning the Owner role to a user, verify the user exists on your {{site.data.var.ece}} account. See [Add a new user][2]. If you need to add the user, but an existing account Owner or Admin cannot help, any Magento user with access to New Relic's [Magento Partnership Owner Account][3] can add users on the behalf of the customer.
```
